### DISCUSSION TITLE
MPTCP在多线网络下的应用？

### DISCUSSION THREAD

**Discussion by lsylsy2** [XTLS/Xray-core#2815] [Q&A]
在1.8.6中开始支持了MPTCP，但是尝试了一些配置之后暂时还不能实现多网聚合的功能。能启用mptcp并在ip mptcp monitor中看到连接建立，但是测速时在第二条宽带上看不到流量也看不到连接。有人有经验分享吗？

**Comment by LiuTangLei** [comment]
负载均衡看到是同一个网址，可能都分配到同一个网卡去了，如果改成平均分配，对其他应用又不友好
